.env.template

```diff
@@ -4,3 +4,18 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY=
+TRELLO_TOKEN=
+TRELLO_BOARD_ID=
+ID_LIST_TODO=
+ID_LIST_DOING=
+ID_LIST_DONE=
+ID_LIST_NEWLIST=
+CARDS_DEVOPSTRELLOBOARD=
+CARDS_TODO_CARD_ONE=
+CARDS_TODO_CARD_TWO=
+CARDS_TODO_CARD_THREE=
+CARDS_TRELLODONE_CARD=
```

Comment:
I can't see any evidence that any of these are used anywhere; if that's the case we should get rid of them